waku/v2/protocol/waku_noise/noise_utils.nim

```diff
@@ -57,6 +57,12 @@ proc pkcs7_unpad*(payload: seq[byte], paddingSize: int): seq[byte] =
   let unpadded = payload[0..payload.high-k.int]
   return unpadded
 
+# Simple utility that checks if the given variable is "default", 
+# Therefore, it has not been initialized
+proc isDefault*[T](value: T): bool = 
+  {.inline.}
```

Comment:
Note that the inline pragma should be between the type and the equal sign: https://nim-lang.org/docs/manual.html#:~:text=proc%20host*(s%3A%20Socket)%3A%20int%20%7B.inline.%7D%20%3D

Reply:
Resolved in [587a037](https://github.com/status-im/nwaku/pull/1104/commits/587a03781ad9f38ef9d1b534294f0e87125c918f)